server/reports/commission/commission-report.js

```diff
@@ -0,0 +1,78 @@
+// module "commission-report"
+const { Parser } = require("json2csv");
+
+
+/* A handy request URLS for testing:
+http://localhost:8080/reports/commission?format=csv&startdate=111&enddate=2222
+*/
+function handleExpressRequest(req, res) {
```

Comment:
This is less of a problem and more of a nit pick, but res isn't terribly meaningful as a name by itself.

Reply:
Agreed. I'll rename them to request and response. The names req and res are pretty standard in the example code I see, but they are not good names.